examples/generate_surrogate_test.py

```diff
@@ -0,0 +1,156 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+"""
+Example of generating a Dynamic Mode Decomposition surrogate model using the battery model 
+"""
+# from prog_models.models import BatteryCircuit as Battery
+# VVV Uncomment this to use Electro Chemistry Model VVV
+# from prog_models.models import BatteryElectroChem as Battery
+from prog_models.models import BatteryElectroChemEOD as Battery
+import numpy as np
+import matplotlib.pyplot as plt
+
+def run_example(): 
+    ### Example 1: Standard DMD Application 
+    ## Step 1: Create a model object
+    batt = Battery()
+
+    ## Step 2: Define future loading functions for training data 
+    # Here, we define two specific loading profiles. These could also be generated programmatically, for as many loading profiles as desired 
```

Comment:
Maybe include (but not use) a commented out example of generating profiles programmatically? Not sure if it's done differently or would break anything but might be helpful seeing how it's done?